src/main/java/seedu/contax/logic/parser/ParserUtil.java

```diff
@@ -238,4 +238,23 @@ public static Duration parseDuration(String duration) throws ParseException {
 
         return new Duration(convertedDuration);
     }
+
+    /**
+     * Parses a {@code String userCommand} into a {@code command with index}.
```

Comment:
```suggestion
     * Parses a {@code String userCommand} into a {@code commandWithIndex}.
```
stuff inside javadoc {@code} should be named like variables

Reply:
changed